models/sale.coffee

```diff
@@ -116,6 +116,10 @@ module.exports = class Sale extends Backbone.Model
   isRegistrationEnded: ->
     @isAuction() and moment().isAfter(@get 'registration_ends_at')
 
+  isLiveAuction: ->
+    @get('live_start_at') and moment(@get('live_start_at')).isBefore(moment(@get('end_at')))
+
+  # Is there a way to manually end/close out a live sale besides looking at end_at?
```

Comment:
Two thoughts--
- Right now, we have a field `ended_at` that we store in gravity, which we generally use to see if a sale has ended. Doesn't look like that's exposed in the json anywhere, probably because we're looking at `end_at` or `live` in most places on the client. We could export and use that if specifically care about it _ending_ vs. _not being open_.
- In this case, the `live` field should also tell us whether or not a sale is open-- when a sale ends we set `live: false` so I think looking for that in combination with being after `live_start_at` might also be sufficient.

Also, I noticed that `isLiveAuction` is very similar to `isLiveOpen`-- maybe there's a way to combine the two.